bin/main.rb

```diff
@@ -0,0 +1,150 @@
+require './../lib/SymbolTB'
+require './../lib/Functions'
+require './../lib/Board'
+
+$debug_on = false
+$player_name = Array[nil,nil]
+$player_weapon = Array[nil,nil]
+$player_title = Array["What do you wanna be called?","You are playing against?"]
+$new_board = [[" "," "," "],[" "," "," "],[" "," "," "]]
+$player_choice = Array.new(2) { Array.new { Array.new(2) } }
+$player_count=Array[0,0];
+$marks_checked = Array[];
+$player_won = -1;
+$last_choice="N/A"
+
```

Comment:
**Global variables** are not allowed.
You should think in a OOP way and create classes containing the Game Logic. Each class with its own attributes.

Reply:
i) player_won its just a flag,
ii) last_choice just a message
iii) marks_checked its player input

Global variable fixed in lastest commit